src/handlers/assign.rs

```diff
@@ -129,11 +129,7 @@ pub(super) async fn parse_input(
     event: &IssuesEvent,
     config: Option<&AssignConfig>,
 ) -> Result<Option<AssignInput>, String> {
-    let config = match config {
-        Some(config) => config,
-        None => return Ok(None),
-    };
-    if config.owners.is_empty() || !event.issue.is_pr() {
+    if config.is_none() || !event.issue.is_pr() {
```

Comment:
Right, this actually meant that even with `[assign.owners]` set, it wasn't possible to `r?` in PR description if the owners were empty, even though it was possible to `r?` in PR comments, that didn't make a lot of sense.